### ISSUE TITLE
Callback for ffmpeg job completion

### ISSUE THREAD

**Issue by mithranalandur** [bramp/ffmpeg-cli-wrapper#19]
It would be nice if the there is callback after the ffmpeg transcoding is complete, since from java we have to have limit the number of  ffmpeg processes running on the system using Executorservice. I have created a pull request , i can also send a patch if required.

**Comment by mandrakey** [comment]
I would even go a little further and add possibilities to track the FFmpeg progress. It is possible by extracting data from the "Duration: " and "time=" output lines.
It would, however, be necessary to track the ffmpeg command output and analyse it all the time...

**Comment by bramp** [comment]
That is a very good suggestion madrakey. At some point I would like to track the output, but for now I don't.

As for this issue in particular, https://github.com/bramp/ffmpeg-cli-wrapper/pull/18 was created, and was limited to tracking when the job ended. We discussed alternatives and ended up rejecting the pull request.

I've created https://github.com/bramp/ffmpeg-cli-wrapper/issues/21 to specifically track parsing output and providing progress.
